framework/src/dirackernels/DiracKernelInfo.C

```diff
@@ -108,7 +108,7 @@ DiracKernelInfo::updatePointLocator(const MooseMesh & mesh)
 }
 
 const Elem *
-DiracKernelInfo::findPoint(Point p, const MooseMesh & mesh)
+DiracKernelInfo::findPoint(Point p, const MooseMesh & mesh, const std::set<SubdomainID> * blocks)
```

Comment:
As @lindsayad mentioned - let's not pass blocks by pointer here.  Should be a const reference.